src/kernels/kernelFinder.ts

```diff
@@ -73,4 +77,9 @@ export class KernelFinder implements IKernelFinder {
 
         return kernels;
     }
+
+    // Give the info for what kernel finders are currently registered
```

Comment:
Also added this so that the UI could see what kernel finders are registered. I didn't just go off the controllers that we see registered since if you add a kernel source with no controllers you would still want to see it as an option in the UI so you know that you connected to the server correctly.